lib/ripper-tags/data_reader.rb

```diff
@@ -130,20 +132,33 @@ def normalize_encoding(str)
 
     def each_tag
       return to_enum(__method__) unless block_given?
+
       file_finder.each_file do |file|
         begin
           $stderr.puts "Parsing file #{file}" if options.verbose
           extractor = tag_extractor(file)
         rescue => err
-          if options.force
-            $stderr.puts "Error parsing `#{file}': #{err.message}"
-          else
+          # print the error
+          $stderr.puts "Error parsing `#{file}': #{err.message}"
+
+          # print sexp and unwrap the RuntimeError
+          if err.is_a?(Visitor::Error)
+            sexp = err.sexp.map { |i| i.is_a?(Array) ? "[...]" : i }
```

Comment:
I don't think it's necessary to print the sexp. Nobody will understand it unless they're literally working on ripper-tags code. I think that would alleviate the need for having Visitor::Error.

It would be great if we could print the line of the file on which the error has ocurred, but that won't be possible without significantly refactoring Parser and Visitor to always preserve line numbers, no matter what happens.